src/main/java/frc/robot/Robot.java

```diff
@@ -192,56 +97,27 @@ public CommandChain LEDCloseBeak() {
         driveController.getButton(XBoxButton.BUMPER_LEFT).whileHeld(drive.rightSlowTurn());
-        driveController.getButton(XBoxButton.START).whenPressed(maflipulator.cancel());
+        driveController.getButton(XBoxButton.BUMPER_LEFT).whileHeld(leds.blinkLights(Leds.fuschia, 1, 1));
         driveController.getButton(XBoxButton.Y).toggleWhenPressed(drive.driveBackwards());
+        driveController.getButton(XBoxButton.B).whenPressed(lift.deployArms());
 
-        // xBoxCoPilot.getButton(XBoxButton.X).whenPressed(manipulator.extendArms());
-        // xBoxCoPilot.getButton(XBoxButton.B).whenPressed(manipulator.retractArms());
-        driveController.getButton(XBoxButton.X).whenPressed(manipulator.extendArms());
-        driveController.getButton(XBoxButton.B).whenPressed(manipulator.retractArms());
-        // xBoxCoPilot.getButton(XBoxButton.A).whenPressed(manipulator.Eject());
-        // xBoxCoPilot.getButton(XBoxButton.Y).whileHeld(manipulator.Intake());
-        // xBoxCoPilot.getButton(XBoxButton.A).whenPressed(manipulator.extendArms());
-        // xBoxCoPilot.getButton(XBoxButton.A).whenPressed(manipulator.extendArms());
-
-        // manipulator.switchTrigger.whileActive(leds.turnOnGreen());
-
-        // xBoxCoPilot.getButton(XBoxButton.A).whileHeld(goToLoadingStation());
-        // xBoxCoPilot.getButton(XBoxButton.A).whenReleased(stowAndGo());
-        // xBoxCoPilot.getButton(XBoxButton.Y).whenReleased(stowAndGo());
-        // xBoxCoPilot.getButton(XBoxButton.Y).whileHeld(goToFloorLoad());
-
-        // // povUp.whenReleased(stowAndGo());
-        // // povUp.whileHeld(goToCargoShipCargo());
-
-        // // povDown.whenReleased(stowAndGo());
-        // // povDown.whileHeld(goToRocketLowCargo());
-
-        // xBoxCoPilot.getButton(XBoxButton.A).whileHeld(maflipulator.pressRotate());
-        // xBoxCoPilot.getButton(XBoxButton.A).whenReleased(stowAndGo());
         xBoxCoPilot.getButton(XBoxButton.B).whenPressed(lift.goToHeight(Heights.kRocketHatchMid));
         xBoxCoPilot.getButton(XBoxButton.X).whenPressed(lift.goToHeight(Heights.kRocketHatchHigh));
-        // xBoxCoPilot.getButton(XBoxButton.B).whenReleased(stowAndGo());
-
-        // povRight.whenReleased(stowAndGo());
-        // povRight.whileHeld(goToRocketMiddleCargo());
-
-        // xBoxCoPilot.getButton(XBoxButton.X).whileHeld(goToRocketHighHatch());
-        // xBoxCoPilot.getButton(XBoxButton.X).whenReleased(stowAndGo());
-        // povLeft.whenReleased(stowAndGo());
-        // povLeft.whileHeld(goToRocketHighCargo());
-
-        // //
-        // xBoxCoPilot.getButton(XBoxButton.STICK_LEFT).whenReleased(stowAndGo());
-        // xBoxCoPilot.getButton(XBoxButton.STICK_LEFT).whenPressed(maflipulator.goToScoringPosition());
+        xBoxCoPilot.getButton(XBoxButton.BUMPER_LEFT).whenPressed(LEDOpenBeak());
+        xBoxCoPilot.getButton(XBoxButton.BUMPER_RIGHT.get()).whenPressed(LEDCloseBeak());
```

Comment:
The `.get()` is unnecessary here, please remove it

Reply:
done